package io.github.anvell.either.resources

internal object TestExceptions {
    object TestErrorOne : Exception()
    object TestErrorTwo : Exception()
}
